gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Util/SocketServerSettings.cs

```diff
@@ -0,0 +1,77 @@
+#region License
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#endregion
+
+using System;
+using System.IO;
+
+namespace Gremlin.Net.IntegrationTest.Util;
+
+public class SocketServerSettings
+{
+    public int PORT { get; set; }
```

Comment:
nit: variable naming

Reply:
Naming is in this style to match the YAML which is in the style of Java constants. I will look into defining a naming convention to convert between the existing yaml and a more idiomatic .NET naming scheme.

Reply:
[example](https://github.com/aaubry/YamlDotNet/blob/master/YamlDotNet.Samples/DeserializeObjectGraph.cs#L100)

Reply:
Thanks this is much better.